src/pom.xml

```diff
@@ -13,7 +13,7 @@
     <module>services</module>
   </modules>
   <properties>
-    <gt.version>30.0</gt.version>
+    <gt.version>31-SNAPSHOT</gt.version>
```

Comment:
version change relates to this PR upstream, it seems: https://github.com/geotools/geotools/pull/4534